src/components/OSCALSystemCharacteristics.js

```diff
@@ -55,7 +55,7 @@ export default function OSCALSystemCharacteristics(props) {
             </Grid>
             <Grid
               item
-              xs={7}
+              xs={12}
```

Comment:
This ends up with a lot of white space to the right of most titles.  Can we just shrink the system name and expand the props?

Reply:
Is this closer to your idea? : 

![image](https://user-images.githubusercontent.com/35196021/127335011-59e43018-1c31-4e28-98a1-faabeef4e406.png)

Reply:
Yes, that looks better IMO.  Maybe give the system name one more grid size and the props one less?  Or do the props start to wrap if you do that?

Reply:
The props appear to still display well with this adjustment. I'll post this fix.

Reply:
The props and system name seem to both fit well in their grid with this change. Will post this fix.